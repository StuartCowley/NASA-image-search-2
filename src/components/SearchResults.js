const SearchResults = () => {
  return (
    <>
      <p>Search results</p>
    </>
  );
};

export default SearchResults;
