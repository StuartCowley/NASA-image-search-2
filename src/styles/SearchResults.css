.search-results {
  margin-top: 30px;
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  justify-content: space-evenly;
  align-items: flex-start;
}

.search-results__image {
  width: 80%;
  border: 3px solid red;
  margin-top: 5px;
}

@media screen and (min-width: 768px) {
  .search-results__image {
    margin: 5px;
    height: auto;
  }
}

@media screen and (min-width: 1024px) {
  .search-results__image {
    margin: 5px;
    height: auto;
    width: 25%;
  }
  .search-results {
    margin-top: 50px;
  }
}
