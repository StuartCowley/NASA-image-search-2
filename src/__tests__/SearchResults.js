import { render } from "@testing-library/react";
import SearchResults from "../components/SearchResults";

describe("SearchResults", () => {
  test("component renders correctly", () => {
    const { asFragment } = render(<SearchResults />);

    expect(asFragment()).toMatchSnapshot();
  });
});
