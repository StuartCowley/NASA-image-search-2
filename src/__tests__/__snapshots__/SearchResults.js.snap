// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResults component renders correctly 1`] = `
<DocumentFragment>
  <p>
    Search results
  </p>
</DocumentFragment>
`;
